rune/libenclave/internal/runtime/pal/skeleton/liberpal-skeleton.c

```diff
@@ -714,6 +727,58 @@ int __pal_init_v1(pal_attr_v1_t *attr)
 
 	parse_args(attr->args);
 
+	if (tls_server) {
```

Comment:
If tls server is a main function, it would be launched in __pal_exec, like how wait_time is acted.